packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -2554,6 +2546,56 @@ describe("Teal Opcodes", function () {
 			});
 		});
 
+		describe("FirstValidTime", function () {
+			this.beforeEach(function () {
+				interpreter = new Interpreter();
+				interpreter.runtime = new Runtime([]);
+				interpreter.runtime.ctx.tx = cloneDeep(TXN_OBJ);
+				interpreter.tealVersion = MaxTEALVersion;
+			})
+
+			it("Should throw exception if txn.lv - txn.fv > 1000 + 1", function () {
+				interpreter.runtime.ctx.tx.fv = 1999; //last valid = 3000, 3000 - (1999 - 1) > 1001
+				const op = new Txn(["FirstValidTime"], 1, interpreter);
```

Comment:
Do we have enum created for this constant string?

Reply:
yes, already updated it. Thanks for noticing